services/brig/src/Brig/API/Handler.hs

```diff
@@ -31,13 +31,13 @@ module Brig.API.Handler
   )
 where
 
-import Bilge (MonadHttp, RequestId (..))
+import Bilge (RequestId (..))
 import Brig.API.Error
 import qualified Brig.AWS as AWS
 import Brig.App
 import Brig.CanonicalInterpreter (BrigCanonicalEffects, runBrigToIO)
 import Brig.Email (Email)
-import Brig.Options (setWhitelist)
+import Brig.Options (setAllowlistEmailDomains, setAllowlistPhonePrefixes)
 import Brig.Phone (Phone, PhoneException (..))
 import qualified Brig.Whitelist as Whitelist
```

Comment:
Maybe worth going all in and changing terms everywhere? :)

Reply:
oh man!  :)

cd42e8d944eddcf75dbe49683e4ff67b7a61334c

also 8c1626bdbb4bd4a47ab1e2121c04b0a69bf25301

what else did i miss?